common/lib/xmodule/xmodule/tests/__init__.py

```diff
@@ -59,6 +61,8 @@ def __init__(self, **kwargs):
         kwargs.setdefault('id_reader', id_manager)
         kwargs.setdefault('id_generator', id_manager)
         kwargs.setdefault('services', {}).setdefault('field-data', DictFieldData({}))
+        self.tracker = DjangoTracker()
+        tracker.register_tracker(self.tracker)
```

Comment:
@mulby @martynjames I had to revert this back to using mock only on emit function because number of tests using this code override explicitly only emit function in tests. If I mock the whole tracker here they all fail.